var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -11,20 +11,21 @@ class NeurodamusModel(Package):
     """An 'abstract' base package for Simulation Models. Therefore no version.
        Eventually in the future Models are independent entities, not tied to neurodamus
     """
-    depends_on('neurodamus-core')
-
     variant('coreneuron',  default=False, description="Enable CoreNEURON Support")
     variant('profile',     default=False, description="Enable profiling using Tau")
     variant('synapsetool', default=True,  description="Enable Synapsetool reader")
     variant('sonata',      default=False, description="Enable Synapsetool with Sonata")
     variant('plasticity',  default=False, description="Use optimized ProbAMPANMDA_EMS and ProbGABAAB_EMS")
+    variant('python',      default=False, description="Install neurodamus-python alongside")
+
+    depends_on('neurodamus-core')
+    depends_on('neurodamus-core+python', when='+python')
 
```

Comment:
Fernando, just to keep things simple I wonder if we have to add python as variant. We just install by default but not announce this to user.

Reply:
You mean for the moment we dont even show the option?

Reply:
its ok...I was thinking if python can be always installed...no variant exposed...but its ok for now.